ucm/sof-glkda7219max/HiFi.conf

```diff
@@ -0,0 +1,106 @@
+SectionVerb {
+
+	EnableSequence [
+		cdev "hw:sofglkda7219max"
+	]
+
+	DisableSequence [
+		cdev "hw:sofglkda7219max"
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker Playback"
+
+	Value {
+		PlaybackPCM "hw:sofglkda7219max,0"
+		PlaybackChannels "2"
+	}
+
+	EnableSequence [
+		cdev "hw:sofglkda7219max"
+		cset "name='Spk Switch' on"
+	]
+
+	DisableSequence [
+		cdev "hw:sofglkda7219max"
+		cset "name='Spk Switch' off"
+	]
+}
+
+SectionDevice."Headphone Playback" {
+	Comment "Headphone Playback"
+
+	Value {
+		PlaybackPCM "hw:sofglkda7219max,1"
+		PlaybackChannels "2"
+	}
+
+	EnableSequence [
+		cdev "hw:sofglkda7219max"
+		cset "name='Headphone Switch' on"
+		cset "name='Headphone Jack Switch' on"
+		cset "name='Playback Digital Switch' on"
+		cset "name='Mixer Out FilterL DACL Switch' on"
+		cset "name='Mixer Out FilterR DACR Switch' on"
+	]
+
+	DisableSequence [
+		cdev "hw:sofglkda7219max"
+		cset "name='Headphone Switch' off"
+		cset "name='Headphone Jack Switch' off"
+		cset "name='Playback Digital Switch' off"
+		cset "name='Mixer Out FilterL DACL Switch' off"
+		cset "name='Mixer Out FilterR DACR Switch' off"
+	]
+}
+
+SectionDevice."Mic" {
+	Comment "Headphone Capture"
+
+	Value {
+		CapturePCM "hw:sofglkda7219max,1"
+		CaptureChannels "2"
+	}
+
```

Comment:
no jack detection information?

Reply:
@plbossart  As I see from `amixer contents`, there is no headset jack detection info. only three HDMI jack detection info, and they are added. 
'Headphone Jack Switch' is not the one we want.

Reply:
it's in the machine driver

````c
	/*
	 * Headset buttons map to the google Reference headset.
	 * These can be configured by userspace.
	 */
	ret = snd_soc_card_jack_new(rtd->card, "Headset Jack",
			SND_JACK_HEADSET | SND_JACK_BTN_0 | SND_JACK_BTN_1 |
			SND_JACK_BTN_2 | SND_JACK_BTN_3 | SND_JACK_LINEOUT,
			&broxton_headset, NULL, 0);


````

Reply:
@plbossart Yes, "Headset Jack" is in the driver, but I am sure it's not in the `amixer contents` (attached), seems we got an issue here.
[glk_amixer.log](https://github.com/thesofproject/alsa-ucm-conf/files/4082220/glk_amixer.log)

Reply:
@juimonen can you help here?